plugins/rspamd.js

```diff
@@ -159,6 +159,13 @@ exports.parse_response = function (rawData, connection) {
         return;
     }
 
+    if (Object.keys(data).length === 1 && data.error) {
+        connection.transaction.results.add(plugin, {
```

Comment:
Would it be preferred to send errors to connection.logerror or connection.transaction.results.add (or both)? Currently the plugin has some mixed behaviour in this respect.